src/eei.cpp

```diff
@@ -743,6 +741,8 @@ inline int64_t maxCallGas(int64_t gas) {
 
       call_message.gas = gas;
 
+       ensureSenderBalance(call_message.value);
```

Comment:
I think we need to wrap this into `if (isZeroUint256(call_message.value)` or `if (import->base == Name("call") || import->base == Name("callCode"))` for explicitness.